packages/yoga/src/Feedback/web/Feedback.jsx

```diff
@@ -40,7 +40,7 @@ function Feedback({ variant, title, description, children, center, ...props }) {
   let primaryButton;
   let secondaryButton;
   let captionElement;
-  let titleElement = <Title>{title}</Title>;
+  let titleElement = <Title as="h1">{title}</Title>;
```

Comment:
This element has already been changed to an `h1` in the PR below.

https://github.com/gympass/yoga/pull/854